### DISCUSSION TITLE
MusicGen+ V1.2.8c

### DISCUSSION THREAD

**Discussion by GrandaddyShmax** [GrandaddyShmax/audiocraft_plus#13] [Announcements]
Change log:

V1.2.8c:
- Implemented Reverse compatibility for audio info tab with previous versions

V1.2.8b:
- Fixed the error when loading default models

V1.2.8a:
- Adapted Audio info tab to work with the new structure prompts feature
- Now custom models actually work, make sure you select the correct base model

V1.2.8:
- Now you will also receive json file with metadata of generated audio
- Added error messages in Audio Info tab
- Added structure prompts: you can select bpm, key and global prompt for all prompts
- Added time display next to each prompt, can be calculated with “Calculate Timings” button

<hr /><em>This discussion was created from the release <a href='https://github.com/GrandaddyShmax/audiocraft_plus/releases/tag/1.2.8c'>MusicGen+ V1.2.8c</a>.</em>

**Comment by pramatias** [comment]
Great stuff! Really good improvements. Json metadata was a must.
